tests/test_compare_with_default_qubit.py

```diff
@@ -121,6 +121,9 @@ def circuit():
                                 observable_pars = [np.array([[1,1j],[-1j,0]])]
                             else:
                                 raise IgnoreOperationException('Skipping in automatic test because I don\'t know how to generate parameters for the observable '+observable+" with par_domain="+str(observable_class.par_domain))
+                        elif str(observable) == "SparseHamiltonian":
```

Comment:
Would it make sense to make this branch more general using `device.supports_observable`? Something like `if dev.supports_observable(observable.__class__)`

Reply:
Yes it would be better! But I let it as it is for the moment, I will check again in the near future.